src/Leela.cpp

```diff
@@ -176,6 +194,109 @@ static void parse_commandline(int argc, char *argv[]) {
         cfg_quiet = true;  // Set this early to avoid unnecessary output.
     }
 
+    if (vm.count("handicap")) {
```

Comment:
This should be inferred from the "handicap" commands rather than being a command line option.

Reply:
That may be the way to go but would exclude handicap games set up by passing the first few moves.

Reply:
Then we add a lz-pass_handicap.